lib/generators/dsfr_component/USAGE

```diff
@@ -0,0 +1,21 @@
+Description:
+    Generates the required structure to implement a new component for the DSFR.
+
+    The NAME needs to be specified as the first parameter, without the
+    component part, so in the example thereafer, Fancy rather than
+    FancyComponent.
+
+    Additional parameters for the compoment can be specified with the
+    --params switch and should be a list of ARG_NAME:TYPE pairs. The
+    ARG_NAME is used as a keyword parameter for the component's
+    initialize method, and the TYPE should be a Ruby class/type
+    because it will be used to fill in the RDoc comments.
```

Comment:
Maybe we should add a comment explaining it won't work with nested components like `Accordion::SectionComponent`

Reply:
Because they need to live within a parent folder? Yes, I can add a note.

Reply:
yes, I don't think the code will handle the module nesting and naming 👍